### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by junichi11**

### CODE REVIEW

**Comment by geertjanw** [comment]
But what's the difference between 'basic' and 'full', shouldn't that be explained too here?

File: README.md [hunk 1]
```diff
@@ -43,15 +43,25 @@ Apache NetBeans is an open source development environment, tooling platform, and
 
 ### Building NetBeans
 
-Build the full project:
+Build the basic project:
```
Comment (junichi11) [inline README.md]: Build the basic project (mainly, JavaSE features):

File: README.md [hunk 2]
```diff
@@ -43,15 +43,25 @@ Apache NetBeans is an open source development environment, tooling platform, and
...
 ```
 $ ant
 ```
+or
+```
+$ ant -Dcluster.config=basic
+```
+Build the full project:
```
Comment (junichi11) [inline README.md]: Build the full project (Including Groovy, PHP, JavaEE/JakartaEE features):

**Comment by junichi11** [comment]
How about the above?

**Comment by geertjanw** [comment]
Great, makes sense.

**Pull request merged**